Funcky.EntityFrameworkCore/Funcky.EntityFrameworkCore.csproj

```diff
@@ -18,6 +19,7 @@
     <PropertyGroup Label="Analyzers">
         <EnableNETAnalyzers>true</EnableNETAnalyzers>
         <AnalysisModeReliability>All</AnalysisModeReliability>
+        <EnablePackageValidation>true</EnablePackageValidation>
```

Comment:
Package validation among other things detects when we remove a target framework :)